src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,43 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final long HONDA_MANUFACTURER_ID = 16L;
+    private static final long BRIO_DRIVER_ID = 5L;
+    private static final long SECOND_BRIO_DRIVER_ID = 6L;
+    private static final long OPEL_ASTRA_ID = 3L;
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        System.out.println(carService.getAll());
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Car brio = new Car();
+        brio.setModel("Honda Brio");
+        brio.setManufacturer(manufacturerService.get(HONDA_MANUFACTURER_ID));
```

Comment:
minimize dependency to magic nubmers, use something like. honda.getId() where possible